common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -109,16 +109,7 @@ function(HTML5Video, Resizer) {
         // starts playing. Just after that configurations can be applied.
         state.videoPlayer.ready = _.once(function() {
             if (!state.isFlashMode() && state.speed != '1.0') {
-                // Work around a bug in the Youtube API that causes videos to
-                // play at normal speed rather than at the configured speed in
-                // Safari.  Setting the playback rate to 1.0 *after* playing
-                // started and then to the actual value tricks the player into
-                // picking up the speed setting.
```

Comment:
@muzaffaryousaf 
When the original code was written, the YouTube API had a bug where, in Safari, you couldn't switch to another speed from the default 1.0x. There was a workaround made which, once the video started playing, set the speed to 1.0x and then immediately set the speed again to a different value. The YouTube API has since been updated, and this workaround, though it still works, became unnecessary code.